ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -152,9 +152,8 @@ impl RawData for RawDataServerWrapper {
         let (tx, rx) = channel(self.data_service_response_channel_size);
         let current_version = match &request.starting_version {
             Some(version) => *version,
-            None => {
-                return Result::Err(Status::aborted("Starting version is not set"));
-            },
+            // Live mode if starting version isn't specified
+            None => (self.in_memory_cache.latest_version().await - 1).max(0), // Smallest version is 0 (genesis)
```

Comment:
I don't think the max is necessary; latest_version returns a u64, so negative numbers are impossible